ServiceFinder.OrderService/Tests/ServiceFinder.OrderService.UnitTest/UnitTests/OrderRequestAppServiceTest.cs

```diff
@@ -0,0 +1,143 @@
+﻿using AutoFixture;
+using AutoMapper;
+using FluentAssertions;
+using NSubstitute;
+using ServiceFinder.Domain.PaginationModels;
+using ServiceFinder.OrderService.Application;
+using ServiceFinder.OrderService.Application.DTOs;
+using ServiceFinder.OrderService.Application.Mapper;
+using ServiceFinder.OrderService.Domain.Enums;
+using ServiceFinder.OrderService.Domain.Events;
+using ServiceFinder.OrderService.Domain.Exceptions;
+using ServiceFinder.OrderService.Domain.Interfaces;
+using ServiceFinder.OrderService.Domain.Models;
+using ServiceFinder.OrderService.Domain.Providers;
+using ServiceFinder.OrderService.Domain.Services;
+
+public class OrderRequestAppServiceTests
+{
+    private readonly IFixture _fixture;
+    private readonly IOrderRequestRepository _orderRequestRepository;
+    private readonly IOrderRequestService _orderRequestService;
+    private readonly IMapper _mapper;
+    private readonly OrderRequestAppService _orderRequestAppService;
+    private readonly IDateTimeProvider _dateTimeProvider;
+    private readonly IDomainEventDispatcher _domainEventDispatcher;
+
+    public OrderRequestAppServiceTests()
+    {
+        _fixture = new Fixture();
+
+        _orderRequestRepository = Substitute.For<IOrderRequestRepository>();
+        _dateTimeProvider = Substitute.For<IDateTimeProvider>();
+        _domainEventDispatcher = Substitute.For<IDomainEventDispatcher>();
+
+
+        var config = new MapperConfiguration(cfg =>
+        {
+            cfg.AddProfile<OrderMappingProfile>();
+        });
+        _mapper = config.CreateMapper();
+
+        _orderRequestService = new OrderRequestService(_orderRequestRepository, _domainEventDispatcher, _dateTimeProvider);
```

Comment:
why do you create a real instance of this service if you do not use it ? mock it

Reply:
fixed